src/batch/src/executor/iceberg_scan.rs

```diff
@@ -295,3 +224,239 @@ impl BoxedExecutorBuilder for IcebergScanExecutorBuilder {
         }
     }
 }
+
+struct PositionDeleteFilter {
+    // Delete columns pos for each file path, false means this column needs to be deleted, value is divided by batch size
+    position_delete_file_path_pos_map: HashMap<String, Vec<Vec<bool>>>,
```

Comment:
I think using `HashMap<String, HashMap<usize, Vec<bool>>,` is more memory efficient.